### DISCUSSION TITLE
Prediction in Ungauged Basins

### DISCUSSION THREAD

**Discussion by Rajiv-0** [neuralhydrology/neuralhydrology#146] [Q&A]
This is a general question. I am working on predicting the streamflow in ungauged basins. Neural Hydrology is good in doing so. I have tried training on several catchments and then predicting on few catchments and it works well (not for all catchments but I understand). I have also tried several combinations of static attributes to check the differences.
 Is there anything I should try to improve the performance of the model? Anything I should look into to get better predictions? I would appreciate any ideas.

**Comment by tberends** [comment]
Hello @Rajiv-0, note sure if you question is still relevant but I would like to share my experiences with you. Indeed the method is Neural Hydrology is useful for PUBs. In my case the use of the MTS-LSTM model was working very well and I set update an experiment scheme with tens of combinations for hyperparameters which provides me the info for to find the best configuration.
